### PR TITLE
Added styled buttons solidbutton and outlinebutton

### PR DESCRIPTION
**Pull request by WeilonYing**
Added two styled buttons: outline button and solid button

Screenshot: 
image

### CODE REVIEW

**Comment by avalonche** [comment]
great buttons weilong.

File: src/components/common/SolidButton/index.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { TouchableOpacity, ActionSheetIOS } from 'react-native';
```
Comment (avalonche) [inline src/components/common/SolidButton/index.js]: What does `ActionSheetIOS` do...

File: src/components/common/OutlineButton/index.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { TouchableOpacity, ActionSheetIOS } from 'react-native';
```
Comment (avalonche) [inline src/components/common/OutlineButton/index.js]: What does `ActionSheetIOS` do...

**Comment by WeilonYing** [comment]
Amended commit to address requested changes

**Pull request merged**